README.md

```diff
@@ -1,2 +1,56 @@
-# phenopacket-scraper-api
+# Phenopacket-Scraper Api
+
 REST API for phenopacket-scraper
+
+##Setup:
+
+To get the project's source code, clone the github repository:
+
+    $ git clone https://github.com/monarch-initiative/phenopacket-scraper-api.git
+
+Install VirtualEnv using the following:
+
+    $ [sudo] pip install virtualenv
+
+Create and activate your virtual environment:
+
+    $ virtualenv venv
+    $ source venv/bin/activate
+
+Install all the required packages:
+
+	$ venv/bin/pip install -r requirements.txt
+
+Create your database and the superuser by running this from the root directory of phenopacketscraper:
```

Comment:
Saying 'root directory' implies (to me) the working directory `phenopacket-scraper-api`. It might be clearer to future readers if you phrase it as:

> ... by running this from the `phenopacketscraper/` subdirectory.